docs/guide/solr.md

```diff
@@ -124,6 +111,7 @@ ez_search_engine_solr:
             core: '%solr_core%'
     connections:
         default:
+            distribution_strategy: standalone
```

Comment:
`standalone` is the default value for the `distribution strategy` so it might be skipped in the example.